pkg/operatorsource/configuring.go

```diff
@@ -105,9 +105,9 @@ func (r *configuringReconciler) Reconcile(ctx context.Context, in *v1.OperatorSo
 	}
 
 	// Now that we have updated the datastore, let's check if the opsrc is new.
-	// If it is, let's force a resync for CatalogSourceConfig.
+	// If it is, let's force a resync for CatalogSources.
 	if isResyncNeeded {
```

Comment:
This whole section of logic seems unnecessary now, right? I don't think this comment is correct, because the resyncer updates the cached app-registry info that the catalogsourceconfig controller uses to know about what metadata was pulled. Since they don't exist anymore, the whole cache sync just isn't needed anymore

Reply:
@kevinrizza that's what I thought too, but when I removed this the catalogsource that was being created from the opsrc was missing the packages it is supposed to have. I discovered that after this test failed when this logic was not there: https://github.com/operator-framework/operator-marketplace/blob/master/pkg/operatorsource/configuring_test.go#L27

Reply:
@kevinrizza turns out that was because of an `Expect()` in the test, which I have removed, so the resync logic has been removed. FYI we still need to `WriteMetaDataToDataStore()`.
Updated PR.